rules-tests/CodeQuality/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/skip_property_on_array_access.php.inc

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\Isset_\IssetOnPropertyObjectToPropertyExistsRector\Fixture;
+
+use ArrayAccess;
+use Enumerable;
+
+/**
+ * @template TKey of array-key
+ *
+ * @template-covariant TValue
+ *
+ * @implements ArrayAccess<TKey, TValue>
+ */
+class Collection implements ArrayAccess, Enumerable {}
+
+class VariableValue {
+    public function __construct(public mixed $value) {}
+}
+
+final class SkipPropertyOnArrayAccess
```

Comment:
As in PSR-4, there can be only 1 class per file. It seems the `Enumerable` class is missing here. I was trying to run code locally with standalone files, but failed.

Could you split the non-fixture classes to `/Source` directory?

Reply:
Sure, Enumerable was not needed for the reproduction

Reply:
Thank you :clap: